src/index.ts

```diff
@@ -100,6 +103,7 @@ export class UnleashClient extends TinyEmitter {
             } 
             resolve();    
         });
+        this.fetch = fetch ?? global.fetch
```

Comment:
Could you please explain why will `global.fetch` work in a browser?

Reply:
Looks like `global` is only supported in Chromium. The correct way would be to use `globalThis`, which is widely supported. For IE support, we should fall back to `window`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis

Reply:
thanks. this aligns with my understanding, was thinking it could be some magic in the build step i was unaware of. Had too look up globalThis, and found a nice writeup for others https://blog.logrocket.com/what-is-globalthis-why-use-it/